wg-administrative/README.md

```diff
@@ -20,7 +20,8 @@ It is important that other working groups have autonomy and agency to fulfill th
 | -------------------------------------------|----------------------|----------------------------| -------- |
 | <img src="https://github.com/groundwater.png" width=100 alt="@groundwater">  | Jacob Groundwater [@groundwater](https://github.com/groundwater) | Member | PT (San Francisco) |
 | <img src="https://github.com/mgc.png" width=100 alt="@mgc">  | Matt Crocker [@mgc](https://github.com/mgc) | **Chair** | PT (San Francisco) |
```

Comment:
Was the idea to replace @mgc by @deanihansen here?

Reply:
@mgc should still be there, adding @deanihansen was a previously approved decision to help balance the wg.